src/components/ProblemsTable.tsx

```diff
@@ -135,18 +167,9 @@ export const ProblemsTable = ({
           Filter: tableFilters.TextFilter(),
           minWidth: 75,
           maxWidth: 250,
-          Cell: (row: CellProps<TableRow>) => (
-            <div
-              className='iui-anchor'
-              onClick={() => {
-                context?.setCurrentAuditInfo({
-                  ...row.row.original,
-                  fileName: row.row.original.fileName ?? getFileNameFromId(row.row.original.fileId),
-                });
-              }}
-            >
-              {row.value}
-            </div>
+          Cell: (row: CellProps<TableRow | TableRow['subRows']>) => (
+            // Hide issue if a subrow
+            <div>{!Object.hasOwn(row.row.original, 'subRows') ? '' : row.value}</div>
```

Comment:
can't we reverse the check like Object.hasOwn(row.row.original, 'subRows') ? row.value :  '' ?